java/com/google/copybara/transform/metadata/MetadataModule.java

```diff
@@ -104,6 +107,17 @@
           + "  - a4321bcde first commit description\n"
           + "  - 1234abcde second commit description\n"
           + "```\n")
+  @Example(title = "Removing description",
+      before = "",
+      code = "metadata.squash_notes(\"Changes for Project Foo:\\n\",\n"
+          + "    show_description = False,\n"
+          + ")",
+      after = "This transform will generate changes like:\n\n"
+          + "```\n"
+          + "Changes for Project Foo:\n\n"
+          + "  - a4321bcde\n"
```

Comment:
Technically this isn't true, but follows the style of the others. It would also show Author. I can fix up that in this PR or in another one?

Reply:
Yeah, I would add the author, because that's the actual output with that example.